build/DownloadDependencies/DownloadDependencies.proj

```diff
@@ -0,0 +1,20 @@
+﻿<Project DefaultTargets="Build" ToolsVersion="Current">	
+  <!-- VS insists on reformatting this file and removing any whitespace whenever it is saved.	
+       So, to preserve the formatting we'll put most of the logic in files that we import. -->	
+  <PropertyGroup>	
+    <ProjectGuid>{3ECCAF9D-3B23-4980-83E2-8ACEE0FC6BEB}</ProjectGuid>	
+  </PropertyGroup>	
+  <Import Project="$(MSBuildThisFileDirectory)CommonMSBuild.targets" />	
+  <Import Project="$(MSBuildThisFileDirectory)CommonProperties.props" />	
+  <Import Project="$(MSBuildThisFileDirectory)JarProcessing.targets" />	
+  <ItemGroup>	
+    <None Include="CommonProperties.props" />	
+    <None Include="JarProcessing.targets" />	
+  </ItemGroup>	
+  <ItemGroup>	
+    <ProjectReference Include="..\tools\ExtractRuleDescFromJson\ExtractRuleDescFromJson.csproj">	
```

Comment:
This needs to be deleted

Reply:
This is just rename of project, cleanup will be part of another PR